src/common/modules/UnicodeFontHandler.js

```diff
@@ -0,0 +1,245 @@
+"use strict";
+
+import * as AddonSettings from "/common/modules/AddonSettings/AddonSettings.js";
+// import * as BrowserCommunication from "/common/modules/BrowserCommunication/BrowserCommunication.js";
+import { isMobile } from "./MobileHelper.js";
+
+import { COMMUNICATION_MESSAGE_TYPE } from "/common/modules/data/BrowserCommunicationTypes.js";
+import { fonts } from "/common/modules/data/Fonts.js";
+
+const caseIds = Object.freeze(["Lowercase", "Uppercase", "Capitalize Each Word", "Toggle Case"]);
+const fontIds = Object.freeze(["Superscript", "Small Caps", "All Small Caps", "Unicase", "separator", "Serif bold", "Serif italic", "Serif bold italic", "Sans-serif", "Sans-serif bold", "Sans-serif italic", "Sans-serif bold italic", "Script", "Script bold", "Fraktur", "Fraktur bold", "Monospace", "Double-struck", "separator", "Circled", "Circled (black)", "Squared", "Squared (black)", "Fullwidth"]);
```

Comment:
Hmm, IDs should usualy be constants. In our case thus an "enum-like" (const and frozen) object. This would be better, IMHO. The advantage then is, if you use them somewhere else, you can just import them there. (And you should ahve auto-completion etc.)

Reply:
They are already constant and frozen, so I am not sure exactly what change you would like me to make.

Reply:
OK, I can put them in the `Fonts.js` file. Note that they will probably need to be localized at some point.